
From: Adrian Bunk <bunk@fs.tum.de>

cciss uses /proc to hook into the SCSI subsystem.  If you do not build
/proc support into your kernel then you should also disable tape support in
the driver.  

Signed-off-by: Adrian Bunk <bunk@fs.tum.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/block/Kconfig |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/block/Kconfig~cciss-congig-dependency-fix drivers/block/Kconfig
--- 25/drivers/block/Kconfig~cciss-congig-dependency-fix	Mon Aug  9 15:42:33 2004
+++ 25-akpm/drivers/block/Kconfig	Mon Aug  9 15:42:33 2004
@@ -166,7 +166,7 @@ config BLK_CPQ_CISS_DA
 
 config CISS_SCSI_TAPE
 	bool "SCSI tape drive support for Smart Array 5xxx"
-	depends on BLK_CPQ_CISS_DA && SCSI
+	depends on BLK_CPQ_CISS_DA && SCSI && PROC_FS
 	help
 	  When enabled (Y), this option allows SCSI tape drives and SCSI medium
 	  changers (tape robots) to be accessed via a Compaq 5xxx array 
_
